test/es-module/test-esm-pkgname.mjs

```diff
@@ -1,20 +1,13 @@
-import { mustCall } from '../common/index.mjs';
-import { strictEqual } from 'assert';
+import '../common/index.mjs';
+import assert from 'node:assert';
 
 import { importFixture } from '../fixtures/pkgexports.mjs';
```

Comment:
While we're making changes in tests, it would be nice if we could start adding comments that describe the expected tested behavior or make use of the `node:test` API and provide descriptive names for the tests.